app/src/main/res/values/strings.xml

```diff
@@ -70,4 +72,7 @@
     <string name="speech_unrecognized">
         We couldn\'t recognize what you said. Please try again.
     </string>
+
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
We should remove this placeholder text. It's not even being used in this in-progress code.